downloads/swan-lake-release-notes/swan-lake-2201.11.0/RELEASE_NOTE.md

```diff
@@ -169,6 +169,10 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 11 (2201.11.0)
 
 ### Improvements
 
+- Replaced `Package.md` with `README.md` as the primary documentation for packages, ensuring a unified documentation format across Ballerina Central and version control platforms. The default `README.md` can be overridden by specifying a preferred file using the `readme` field in the [package] table of the `Ballerina.toml`.
```

Comment:
```suggestion
- Replaced `Package.md` and `Module.md` with `README.md` as the primary documentation for packages, ensuring a unified documentation format across Ballerina Central and version control platforms.
```

Reply:
Let's provide an example of ld and new package structure along with `bal build -t lib <package>` highlighting the generation of `README.md` now.